test/test_utils.go

```diff
@@ -281,3 +287,30 @@ func verifySignature(ctx context.Context, t *testing.T, c *clients, obj objects.
 		}
 	}
 }
+
+// restartChainsControllerPod restarts the pod running Chains
+// it then waits for a given timeout for the pod to resume running state
+func restartChainsControllerPod(c kubernetes.Interface, timeout time.Duration) error {
+	pods, err := c.CoreV1().Pods("tekton-chains").List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	pod := pods.Items[0]
+	podUid := pod.UID
+	err = c.CoreV1().Pods(pod.Namespace).Delete(context.Background(), pod.Name, metav1.DeleteOptions{})
```

Comment:
May want to set `GracePeriodSeconds: 0` to speed up the delete.

Reply:
Done!